app/models/order.js

```diff
@@ -20,7 +21,12 @@ export default Model.extend({
   detailType:       attr('string'),
   districtId:       attr('number'),
   ordersPurposes:     hasMany('ordersPurpose', { async: false }),
+<<<<<<< HEAD
   beneficiaryId: attr('string'),
+=======
```

Comment:
need to resolve this

Reply:
Yupp, my bad, missed this. will do it right away. Thanks @steveyken